settings.go

```
@@ -104,6 +117,63 @@ func usage() {
 	os.Exit(2)
 }
 
+func readAndUpdateConfig(body io.ReadCloser) []byte {
+	decoder := json.NewDecoder(body)
+
+	var t AppSettings
+	err := decoder.Decode(&t)
+	if err != nil {
+		return []byte("Error while updating flags via POST request.")
+	}
+	Settings = t
+	newConfig, err := json.Marshal(Settings)
+	if err != nil {
+		return []byte("Error while updating flags via POST request.")
+	}
+	err = viper.ReadConfig(bytes.NewBuffer(newConfig))
+	if err != nil {
+		return []byte("Error while updating flags via POST request.")
+	}
+	return nil
+}
+
+func updateConfig(respBody []byte) {
+	var t AppSettings
+	if err := json.Unmarshal(respBody, &t); err != nil {
+		return
```

why did you ignore errors?